src/views/bouquets/BouquetEditView.vue

```diff
@@ -23,27 +21,136 @@ const steps = [
   'Composition du bouquet',
   'Récapitulatif'
 ]
+const thematiques = [
+   'Produire',
+   'Se nourrir',
+   'Se loger',
+   'Se déplacer',
+   'Préserver',
+   'Consommer',
+   'Chantiers transverses'
+]
+const selectedThematique = ref('Produire')
+const selectedSubTheme = ref('Nucléaire')
```

Comment:
Move to config (cf comment about default values below)

Reply:
+1

Reply:
Okay I did it